modules/apps/app-builder/app-builder-test/src/testFunctional/paths/FormViewBuilder.path

```diff
@@ -42,7 +42,15 @@
 
 <tr>
 	<td>FIELD_LABEL</td>
+<<<<<<< HEAD
+<<<<<<< HEAD
 	<td>//p[text()='${fieldLabel}']</td>
+=======
+	<td>//div[@class='ddm-drag']//p[@class='ddm-label'][contains(text(),'${fieldLabel}')]</td>
+>>>>>>> LRQA-53878 Add paths for new AppBuilder Delete Form View Field from Object test
+=======
+	<td>//p[text()='${fieldLabel}']</td>
+>>>>>>> LRQA-53878 Update Path FIELD_LABEL to match updated testfix
```

Comment:
Hey @tainguyen100, could you please solve this issue? I think it was caused when you were resolving the conflict.